libr/core/cmd_anal.c

```diff
@@ -1790,6 +1814,8 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 			}
 		} else if (fmt == 's') {
 			totalsize += op.size;
+		} else if (fmt == '*') {
+			// TODO: ao* useful for wat? wx [bytes] ?
```

Comment:
maybe add this message as a eprintf? Just to warn user that this feature is not implemented yet.